linkerd/app/core/src/serve.rs

```diff
@@ -39,39 +39,38 @@ pub async fn serve<M, S, I, A>(
                     };
 
                     // The local addr should be instrumented from the listener's context.
-                    debug_span!("accept", client.addr = %addrs.param()).in_scope(|| {
-                        let accept = new_accept.new_service(addrs);
+                    let span = debug_span!("accept", client.addr = %addrs.param()).entered();
+                    let accept = new_accept.new_service(addrs);
```

Comment:
I assume the `entered` here means this span applies for the `new_service` call?

Reply:
yup, that's correct! i thought this was nicer than the previous version, especially since it means that the `spawn` call will be inside that span (which doesn't really matter currently, but might be nice for console purposes eventually)...